docs/DeveloperGuide.md

```diff
@@ -106,28 +108,27 @@ The `Model`,
 * exposes an unmodifiable `ObservableList<Person>` that can be 'observed' e.g. the UI can be bound to this list so that the UI automatically updates when the data in the list change.
 * does not depend on any of the other three components.
 
-
 <div markdown="span" class="alert alert-info">:information_source: **Note:** An alternative (arguably, a more OOP) model is given below. It has a `Tag` list in the `AddressBook`, which `Person` references. This allows `AddressBook` to only require one `Tag` object per unique `Tag`, instead of each `Person` needing their own `Tag` object.<br>
```

Comment:
Seems like markdown isn't converting \*\*Note:** over here to bold. I think it's because it's embedded inside HTML. Try using \<strong>Note:\</strong> instead.

Reply:
Fixed